agent/src/agent/cli/backup.py

```diff
@@ -0,0 +1,23 @@
+import click
+import os
+
+from datetime import datetime
+from agent.modules.constants import AGENT_DB_USER, AGENT_DB, BACKUP_DIRECTORY, AGENT_DB_HOST
+
+
+@click.command()
+def backup():
+    filename = os.path.join(BACKUP_DIRECTORY, f'{AGENT_DB}_{datetime.now():%Y-%m-%d_%H:%M:%S}.dump')
+    if os.system(f'pg_dump -Fc -h {AGENT_DB_HOST} -U {AGENT_DB_USER} {AGENT_DB} > {filename}') == 0:
+        click.secho(f'{AGENT_DB} database successfully dumped to {filename}', fg='green')
+
+
+@click.command()
+@click.argument('dump_file')
```

Comment:
https://click.palletsprojects.com/en/7.x/arguments/#file-arguments

Reply:
I'll get file handle if I use it, but I need only a path to a file